kivymd/uix/button.py

```diff
@@ -1564,7 +1564,7 @@ class BaseFloatingLabel(
 
 
 class MDFloatingBottomButton(BaseFloatingBottomButton):
-    pass
+    ref_name = StringProperty()
```

Comment:
This is not a very good name for a parameter. This name does not reflect the essence.

Reply:
its is not a parameter its just to reference with code itself

Reply:
anyways i ll change to text